### PR TITLE
Update article.md

### PR DESCRIPTION
**Pull request by matmont**

### CODE REVIEW

File: 1-js/02-first-steps/12-nullish-coalescing-operator/article.md [hunk 1]
```diff
@@ -153,14 +153,14 @@ alert(x); // 2
 
 ## Summary
 
-- The nullish coalescing operator `??` provides a short way to choose the first "defined" value from a list.
+- Il nullish coalescing operator `??` fornisce una breve strada per la scelta del primo valore "definito" da una lista di valori.
```
Comment (mean2me) [inline 1-js/02-first-steps/12-nullish-coalescing-operator/article.md:156]: I'd suggest "breve strada" => "scorciatoia"

**Review by mean2me — COMMENTED** [review]
Just a very small and optional change proposed. For me it's good anyway.

File: 1-js/02-first-steps/12-nullish-coalescing-operator/article.md (hunk 1 above)
Comment (matmont) [reply 1-js/02-first-steps/12-nullish-coalescing-operator/article.md:156]: Yeah sure, I agree!

**Comment by longo-andrea** [comment]
The article is ok!  But, none of the titles are translated 😔

**Review by longo-andrea — APPROVED** [review]

**Pull request merged**